app/scripts/constants.js

```diff
@@ -85,7 +85,7 @@ window.OPENSHIFT_CONSTANTS = {
   ],
 
   ENABLE_TECH_PREVIEW_FEATURE: {
-    pipelines: true
+    catalog_landing_page: false
```

Comment:
@sspeiche any preference on what we call this thing, "catalog" seemed to generic since we already have a "catalog" in the console.   Keep in mind this would show up in the docs for how to turn it on for tech previewing.